README.md

```diff
@@ -1572,6 +1573,33 @@ Cluster membership updated in the ProxySQL database!
 ```
 The weights corresponding to the node `192.168.56.33:3306` in the writer-config hostgroups has been updated to the new value `1111`. Note that only writer-config hostgroup has been updated since the node doesn't have a corresponding entry in the writer hostgroup.
 
+### Extra options
+-------------------
+
+__i) -- write-node__
+This option is used to choose which node will be the writer node when the mode is _singlewrite_.  This option can be used with __--enable__ and __--update-cluster__.
+
+A single IP address and port combination is expected.
+
+If --write-node is used, the writer node is given a weight of 1000000 (the default
+weight is 1000).
+
+__ii) --force__
+
+  This will skip existing configuration checks with __--enable__ option in mysql_servers, mysql_users and mysql_galera_hostgroups tables.
```

Comment:
Not clear.
Does force require --enable or can it be used with --enable. What happens if used by itself?

Reply:
`--force` can be used only with `--enable` i.e while setting up proxysql.

`--force` cannot be used by itself, as the PSA script allows at least one of the below options to be present
```
--enable, --disable, --update-cluster, --is-enabled, --adduser, --syncusers, --sync-multi-cluster-users, or --update-mysql-version
```

Reply:
OK. Thanks.

Reply:
You and I must communicate more, please. 
The listing for --force from a couple of weeks ago was just --enable and --update-cluster. 
Changing the option requirements also means I must update the docs.  
By the way - the README only lists --enable and --update-cluster. 
Users become frustrated when they go by the instructions and it doesn't work as described. If it doesn't work, they won't use it.